### DISCUSSION TITLE
Assistance for CLI commands

### DISCUSSION THREAD

**Discussion by nuclearedc** [comictagger/comictagger#620] [Q&A] [answered]
Hi,

I am not very good with CLI commands and I am struggling to tag multiple cbz files in a folder.
Following the wiki, I was able to successfully tag a single file and change the publisher metadata.
Starting cmd in Windows 11
Typing the following command
_"C:\Program Files\ComicTagger\comictagger.exe" -s -t cr -m "publisher=tagging test" -v "C:\Books\Manga\Series Name\Series Name V01 (2022).cbz"_
It works and get the publisher tagged with "tagging test"

I would like to tag genres, atlernateseries and other metadata besides "series", "title", "publisher"... but I cannot guess the proper way to do it.
I tried "genre=Drama^, Action" but I get an error saying that 'genre' is not a valid tag name. same thing for alternateseries.
I tried "credit=genre:Drama^, Action" similar to the solution in the other discussion about penciller and author tagging but without any success.

I also cannot figure out how to tag multiple files within the "series name" folder.
I tried "C:\Books\Manga\Series Name\*.cbz" but I get an error message that the file can't be found. I am not sure if I use quotation marks or the * correctly.

Any help would be appreciated.
Thank you

**Comment by lordwelch** [comment]
For manual updates via the cli [comicbox] is a better option.

The exact set of options available depends on the version that you are using but they don't change very often. Here is the current list of attributes:
```plain
series
series_aliases
issue
issue_count
title
title_aliases
volume
volume_count
genres
description
notes

alternate_series
alternate_number
alternate_count
story_arcs
series_groups

publisher
imprint
day
month
year
language
country
web_link
format
manga
black_and_white
maturity_rating
critical_rating
scan_info

tags
pages
page_count

characters
teams
locations
credits

price
is_version_of
rights
identifier
last_mark
```

Note that these attributes do not work for all tag types and depending on the tag type only a few credit roles are allowed. The ComicRack tags only supports: writer, penciller, inker, colorist, letterer, cover and editor for credits


[comicbox]: https://github.com/ajslater/comicbox

**Reply by nuclearedc** [reply]
Thank you very much for the reply.
I was trying to match the ComicInfo.xml v2.0 schema and its attributes.

I tried "genres" instead of "genre" and it successfully tagged the file.

If you have any recommendation on how to batch tag all the cbz files in a folder, it would greatly help.
The reason for me to learn CLI commands is in order to batch tag genres to the multiples volumes of the same series (Comic Vine does not have genres, but it is quite a useful tag since you can filter your collection by genre in Kavita).
I saw that there was a request made to have that option to batch tag in the GUI and I hope it gets implemented one day as it would be great for someone like me who does not know how to properly write commands.

I tried to look into "comicbox" but without an executable file, I have no clue how to even run it on my Windows 11 computer.
My computer skills are pretty inexistant.

**Reply by lordwelch** [reply, marked as answer]
Use the `-R` option and give it the name of the folder instead of the name of the cbz

**Reply by nuclearedc** [reply]
Excellent!
Thank you so much.
